pkg/util/utils.go

```diff
@@ -34,14 +34,10 @@ import (
 	"k8s.io/minikube/pkg/minikube/out"
 )
 
-// ErrPrefix notes an error
-const ErrPrefix = "! "
-
-// OutPrefix notes output
-const OutPrefix = "> "
-
 const (
 	downloadURL = "https://storage.googleapis.com/minikube/releases/%s/minikube-%s-amd64%s"
+	OutPrefix   = "> " // OutPrefix notes output
```

Comment:
For readability, please move the comment to appear on it's own line before the variable.

Reply:
fixed

Reply:
alright, fixed.